docs/getting-started/clients/mobile/android/index.md

```diff
@@ -58,11 +58,78 @@ To set up a new virtual Android device for debugging:
 
 ![Android SDK configuration](android-sdk.png)
 
-## Testing and Debugging using the Android Emulator
+## Testing and Debugging
+
+### Using the Android Emulator
 
 In order to access `localhost:<port>` resources in the Android Emulator when debugging using Xamarin
 studio on your mac natively, you'll need to configure the endpoint addresses using
 `<http://10.0.2.2:<port>`\> in order to access `localhost`, which maps the Android proxy by design.
 
 [xamarin-vs]: https://learn.microsoft.com/en-us/xamarin/android/get-started/installation/android-sdk
 [android-studio]: https://developer.android.com/studio/releases/platforms
+
+### Using Reverse Proxy
+
+Since your physical device and server are not directly running on the same machine, it's more
+complex to connect them. Although, there is an easy way to solve this problem, connect your mobile
+deployment (Emulator or Device) to your local server by creating a reverse proxy for the `api` and
+`identity` using [ngrok](https://ngrok.com/). ngrok will create public endpoints for your localhost
+server that you can connect to with your mobile app.
+
+1. install `ngrok`, can be done using brew:
+
+```ngrok
+brew install --cask ngrok
+```
```

Comment:
Just something to note, if you install `ngrok` from their site and sign up with an account, it only lets you use one instance at a time and you have to nuke it along with an auth file buried within hidden folders. Installing with brew gets around this, so maybe a callout about that would be good!

Reply:
That is a very good catch i'll add it!